Cisco ASA parsing rules - iso8601 (#25766)
demisto/content: Packs/CiscoASA/README.md

3. When configuring the Syslog Collector, set:
   - vendor as vendor<- Cisco
   - product as product<- ASA

### config timeformat on Cisco ASA

> The only supported date format is RFC 5424, an example: "2023-04-09T16:30:00Z"

Reviewer: Please add a comment that if the user will use different timestamp formant time extraction and mapping will not be supported.

Reviewer 2: Add the following:
Note: If a different timestamp format is used, time extraction and mapping will not be supported.

3. When configuring the Syslog Collector, set:
   - vendor as vendor<- Cisco
   - product as product<- ASA

> ### config timeformat on Cisco ASA

Reviewer 2: ```suggestion
### Configure Timestamp on Cisco ASA
```

> In Cisco ASA product we support only iso8601 date format with timezone.

Reviewer 2: ```suggestion
In Cisco ASA product we support only ISO8601 date format with the time zone.
```

an example ofr the timestamp: "2023-04-09T16:30:00-07:00"

> 1. Access the Cisco ASA command-line interface (CLI) using a console connection or SSH session.

Reviewer 2: ```suggestion
1. Access the Cisco ASA CLI using a console connection or SSH session.
```

2. Enter privileged EXEC mode by typing ```enable``` and entering the enable password.
3. Enter global configuration mode by typing ```conf t```.

> 4. Enter the logging timestamp command to update the timestamp format to iso8601 by typeing

Reviewer 2: ```suggestion
4. Enter the logging timestamp command to update the timestamp format to ISO8601 by typing
```
